client/src/main/java/io/getstream/chat/android/client/utils/SystemTimeProvider.kt

```diff
@@ -1,5 +1,9 @@
 package io.getstream.chat.android.client.utils
 
 class SystemTimeProvider {
-    fun provideTime() = System.currentTimeMillis()
+    fun provideTime() = System.currentTimeMillis() / MILLIS_TO_SECONDS_FACTOR
```

Comment:
What do you think to change the name of this method?
```suggestion
    fun provideCurrentTimeInSeconds() = System.currentTimeMillis() / MILLIS_TO_SECONDS_FACTOR
```

Reply:
Agree